torch/testing/_internal/common_methods_invocations.py

```diff
@@ -145,6 +145,104 @@ def _getattr_qual(obj, name, default=_NOTHING):
+#   automated test coverage by consolidating these lists into one structured and
+#   documented list.
+#
+# Most tests that use every OpInfo are in test_ops.py. More correctly,
+#   test_ops.py contains "test templates" which are instantiated by
+#   PyTorch's "device generic test framework". See the developer Wiki article
+#   above for more information on this test instantiation process.
+#
+# In addition to the OpInfo base class, there are subclasses of OpInfos, like
+#   UnaryUfuncInfo, that extend the base class's supported metadata for
+#   a group of operator with a common structure. For example, the
+#   UnaryUfuncInfos have a "ref" (short for "reference") attribute that defines
+#   a NumPy or SciPy reference for them. These subclasses may have additional
+#   tests in their corresponding files. UnaryUfuncInfos, for example, have
+#   specialized tests in test_unary_ufuncs.py. In the future BinaryUfuncInfos
+#   and ReductionInfos will be created and specialized tests added for them,
+#   too.
+#
+# When an OpInfo is added tests for it are automatically generated based on
+#   its metadata and the templated tests defined in PyTorch's test suite
+#   that use the @ops decorator. Again, for more on this process see the
+#   linked developer Wiki article. Knowing what, exactly, is tested when
+#   adding an OpInfo requires reading the corresponding test files, but
+#   test_ops.py, where the majority of tests live, can be thought of as
+#   validating the following (as of this writing):
+#
+#     1) That an operator's function, method, and inplace variants (if they
+#          exist) perform the same computation.
+#     2) That an operator's aliases, if any, perform the same computation.
+#     3) That an operator implements its gradient and second-order gradient
+#          correctly.
+#     4) That an operators works when traced or scripted by the jit.
+#     5) That operators supporting out= do so correctly.
+#     6) That operators specify metadata, like which dtypes they support,
+#          properly.
+#
+# Note that none of these tests validate the operator actually computes what
+#   it's supposed to. As mentioned previously, OpInfo-based tests can
+#   typically only treat operators as "black boxes." Only in special cases, like
+#   with the UnaryUfuncInfos, have tests been developed that understand the
+#   structure of those operations and that can validate they compute as
+#   as expected.
+#
+# When implementing a new operator, then, an OpInfo should be added, but
+#   (unless the operator being added is in a special class with tests that
+#   can validate its computation) additional tests still need to be added
+#   to verify the operator computes what it's expected to. Even though there
+#   are many linalg ops with OpInfos, for instance, there are still
+#   comprehensive handwritten tests for these operators in test_linalg.py.
+#   What doesn't need to handwritten testing are the propertes listed above,
```

Comment:
```suggestion
#   What doesn't need to be handwritten testing are the properties listed above,
```

Reply:
@anjali411, that suggested revision seems to convey incorrect information.

What about `The properties listed above don't require handwritten tests`?

Reply:
I'll tweak it; good grammar catch